src/program/config/set_operator_state/README

```diff
@@ -0,0 +1,19 @@
+Usage: snabb config set-operator-state [OPTION]... ID ALARM-ID OPERATOR-STATE ALARM-TEXT
+Adds a new operator-state in an alarm.
+
+Available options:
+  -s, --schema SCHEMA        YANG data interface to request.
+  -r, --revision REVISION    Require a specific revision of the YANG module.
+  -f, --format               Selects output format (yang or xpath). Default: yang.
+      --print-default        Forces print out of default values.
+  -h, --help                 Displays this message.
+
+Given an instance identifier, an alarm id and an operator-state, add a new
+operator-state to the given alarm.
+
+Typical usage:
+
+$ snabb config get-state lwaftr external-interface,arp-resolution ack.
```

Comment:
This example uses `get-state`.  Also what's up with the comma and the period?

Reply:
Set-operator-state needs two params: an alarm key and an operator-state.

The alarm key is composed by three parameters: resource, alarm-type-id and alarm-type-qualifier. So I separate each of them by a comma (alarm-type-qualifier is blank in this example). As for the period it was the end of line period (as it's a command line example shouldn't be there).